design/parameters.md

```diff
@@ -0,0 +1,121 @@
+# Loading parameters from external files
+Currently, parameters passed to templates must be declared inline. Here is an example of how one would implement, for example, an expansion of jobs: 
+
+```yaml
+# groups-template.yaml
```

Comment:
This snippet does not work for me and throws the error: Mapping values are not allowed in this context (row 15 of this file) - could you please check that all snippets here are valid?